src/DocBlock/DocBlock.php

```diff
@@ -162,6 +162,36 @@ public function getContent()
         return implode($this->lines);
     }
 
+    /**
+     * Find the line number of the line containing the end of the short
+     * description, if present.
+     *
+     * @return null|int
+     */
+    public function findShortDescriptionEnd()
```

Comment:
For now, this class was exposing informations from docblock text "as is", this is the first method that actually provide some analysis logic. 
For that, I'm not feeling 100% sure that this class is proper place for it, especially as it means that we are extending our BC promise to end users who are using our repo as library.
I understand the need of extraction and for sure it's way better than copy-pasting between fixers, but please extract it to dedicated, standalone class.
We already had that discussion previously for tokenizer. We have `Tokens\Analyzer` namespace with "random", standalone analysis that are internal and which are not covered by any BC promise, while main `Tokens` class is. Similar approach could be done here. Dont be afraid of small classes ;)

Reply:
I get the backward compatibility concern. I've created `ShortDescription` class with `@internal` and `final` keyword.

Reply:
thanks, composition looks a lot better now :+1:  :)